snowflake/src/main/java/org/apache/iceberg/snowflake/SnowflakeCatalog.java

```diff
@@ -214,6 +214,12 @@ public boolean dropNamespace(Namespace namespace) {
         "SnowflakeCatalog does not currently support dropNamespace");
   }
 
+  @Override
+  public boolean dropNamespace(Namespace namespace, boolean cascade) {
```

Comment:
shouldn't be needed

Reply:
I may be missing something, its needed to implement the interface though?
```
Class 'SnowflakeCatalog' must either be declared abstract or implement abstract method 'dropNamespace(Namespace, boolean)' in 'SupportsNamespaces'

Reply:
Oh above you would add this with a "default" throw unsupported operation exception to the api. That way you don't have to add exception throwing code to all the catalogs that don't implement it

Reply:
This also prevents the change from breaking other catalogs not in the Iceberg repo

Reply:
ah! I see what you mean, makes sense. Thanks

Reply:
@RussellSpitzer made it default.